api/src/database/helpers/schema/dialects/cockroachdb.ts

```diff
@@ -11,14 +11,6 @@ export class SchemaHelperCockroachDb extends SchemaHelper {
 		await this.changeToTypeByCopy(table, column, options, (builder, column) => builder[type](column));
 	}
 
-	async changeToText(
-		table: string,
-		column: string,
-		options: { nullable?: boolean; default?: any } = {}
-	): Promise<void> {
-		await this.changeToTypeByCopy(table, column, options, (builder, column) => builder.text(column));
-	}
-
 	async changeToString(
```

Comment:
Didn't replace it initially because it's slightly different in that it'll use the `length` in options, but I've replace them now and modified the `changeToType` to use length when it's string 👍